package.json

```diff
@@ -54,6 +55,8 @@
 		"@nextcloud/browserslist-config": "^2.3.0",
 		"@nextcloud/eslint-config": "^8.2.0",
 		"@nextcloud/stylelint-config": "^2.3.0",
-		"@nextcloud/webpack-vue-config": "^5.4.0"
+		"@nextcloud/webpack-vue-config": "^5.4.0",
+		"node-sass": "^8.0.0",
```

Comment:
Should be replaces with the `sass` package due to https://sass-lang.com/blog/libsass-is-deprecated